specification/apimanagement/control-plane/readme.md

```diff
@@ -29,6 +29,15 @@ openapi-type: default
 tag: package-2017-03
 ```
 
++## Suppression
```

Comment:
The procces to add supressions is different, please remove it.

Reply:
You can start here :)
https://github.com/Azure/adx-documentation-pr/wiki/Swagger-Validation-Errors-Suppression
It should be added in a different PR

Reply:
removed the suppression, and model validator is failing not because of any files in this pr. Look for apimanagement/control-plane